src/osuTK.iOS/iOSGameView.cs

```diff
@@ -712,6 +712,14 @@ public override void LayoutSubviews()
             }
         }
 
+
+        public void OnFileDrop(string s)
```

Comment:
this is never being called..

Reply:
it's getting called here for the first time to handle FileDrops: [ppy/osu-framework#2048 -> GameAppDelegate.cs](https://github.com/ppy/osu-framework/blob/d92ee47e1207951b2490d63e2a0ead1123914f1c/osu.Framework.iOS/GameAppDelegate.cs#L64)